xero/auth.py

```diff
@@ -318,8 +318,11 @@ class PartnerCredentials(PublicCredentials):
 
         >>> rsa_key = "-----BEGIN RSA PRIVATE KEY----- ..."
 
-     2) You'll need to pass a tuple to the Entrust certificate pair.
-
+     2) client_cert is no longer needed regarding new Xero security policy, 
+        but stays for now to be backward-compatible.
+        If you are still implementing Entrust Certificates you'll need 
```

Comment:
I don't think this comment is completely right. If people are using `pyxero` after this merge, they can't use anymore Entrust certificates pair. I think the first sentence should be something more like: `client_cert is no longer used regarding new Xero security policy, but stays for now for backward-compatibility.` and remove `If you are still [...] pair.`